packages/core/src/external/fhir/consolidated/hydrate.ts

```diff
@@ -30,12 +34,11 @@ export async function hydrate({
   if (metadata) {
     metrics.properties = {
       ...metrics.properties,
-      duration,
+      hydrateDurationMs: duration,
```

Comment:
This requires changes on Posthog's dashboards, no? If so, can we flag those changes on the release plan, please?